iterators.tex

```diff
@@ -6349,8 +6356,8 @@
 \oldtxt{\tcode{begin} and \tcode{end} are required to be amortized constant time.}
 
 \pnum
-\newtxt{Given an object \tcode{t} of type \tcode{T}, \oldtxt{the type \tcode{T} models}
-\tcode{Range\newtxt{<T>()}} \newtxt{is satisfied} if and only if}
+\newtxt{Given a value \tcode{t} of type \tcode{T}, \tcode{Range<T>()} is satisfied
+if and only if}
```

Comment:
That works, I think.

Also, I would strike the bullet about `SizedRange`. It doesn't seem appropriate to me. Are you trying to disallow `Ranges`s for which `size(r)` compiles but does the wrong thing? I think it's fine if such a type satisfies `Range` but not `SizedRange`.